rotkehlchen/chain/substrate/typing.py

```diff
@@ -13,14 +13,22 @@
 class KusamaNodeName(Enum):
     """Public nodes for Kusama.
     """
-    PARITY = 0
+    OWN = 0
+    PARITY = 1
```

Comment:
What happened to the web3 node?

Reply:
Only works under web sockets and from time to time... sometimes SSL problems, sometimes connection closed. Parity is the most reliable one using the `SubstrateInterface`.